R/addins.R

```diff
@@ -0,0 +1,8 @@
+addin_lint <- function () {
+  filename <- rstudioapi::getActiveDocumentContext()
+  lintr::lint(filename$path)
+}
+
+addin_lint_package <- function () {
+  lintr::lint_package()
```

Comment:
Probably would be safer to retrieve the project directory (`rstudioapi::getActiveProject()`) and pass that to `lint_package()`?

Reply:
Yep. Indeed I was thinking I completely forgot defensive programming. I'll add this and some more checks and then update the pull request